tests/tests_unit/test_approval_modules_snapshots/my_example_module.yaml

```diff
@@ -82,13 +82,15 @@ TransformationSchedule:
   interval: 7 * * * *
   isPaused: true
 Workflow:
-- externalId: wf_baz
+- description: A workflow for processing data
+  externalId: wf_baz
```

Comment:
I have a preference for always having externalId as the first variable, so if it's all the same to you...?